adapter/internal/oasparser/model/api_operation.go

```diff
@@ -69,9 +106,13 @@ func (operation *Operation) GetID() string {
 }
 
 // NewOperation Creates and returns operation type object
-func NewOperation(method string, security []map[string][]string, extensions map[string]interface{}) *Operation {
+func NewOperation(method string, security []map[string][]string, extensions map[string]interface{}, 
+	mockedAPIConfig MockedAPIConfig) *Operation {
 	tier := ResolveThrottlingTier(extensions)
 	disableSecurity := ResolveDisableSecurity(extensions)
 	id := uuid.New().String()
-	return &Operation{id, method, security, tier, disableSecurity, extensions}
+	if reflect.DeepEqual(MockedAPIConfig{},mockedAPIConfig) {
+		return &Operation{id, method, security, tier, disableSecurity, extensions, MockedAPIConfig{}}
```

Comment:
Because we are still returning an empty struct, we decided to remove this if condition right?